servicex_app/servicex/transformer_manager.py

```diff
@@ -113,20 +113,17 @@ def create_job_object(request_id, image, rabbitmq_uri, workers,
                 field_path="metadata.name"))
         env += [client.V1EnvVar("POD_NAME", value_from=pod_name_value_from)]
 
-        # provide pods with level and logging server info
```

Comment:
Where does this come from now?

Reply:
for some reason this part of code was duplicated. I just removed one copy of it.